### DISCUSSION TITLE
Complete the lyrics 🎤

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#652] [Galaxy Trivia] [answered]
```
When you were here before
Couldn't ________ you in the eye

You're just like an ________
Your ________ makes me cry
You float like a ________
In a ________ world

I wish I was special
You're so ________ special

But I'm a ________
I'm a ________
What the hell am I doin' here?
I don't ________ here
```

**Comment by peterbaccaro** [comment, marked as answer]
When you were here before
Couldn't **LOOK** you in the eye

You're just like an **ANGEL**
Your **SKIN** makes me cry
You float like a **FEATHER**
In a **BEAUTIFUL** world

I wish I was special
You're so **FUCKIN'** special

But I'm a **CREEP**
I'm a **WEIRDO**
What the hell am I doin' here?
I don't **BELONG** here

**Comment by FleetAdmiralJakob** [comment]
When you were here before
Couldn't look you in the eye
You're just like an angel
Your skin makes me cry
You float like a feather
In a beautiful world

I wish I was special
You're so fuckin' special

But I'm a creep
I'm a weirdo
What the hell am I doing here?
I don't belong here.
